tests/dependencies_test.go

```diff
@@ -99,10 +99,19 @@ func installWordpressBundle(p *porter.TestPorter) (namespace string) {
 	return namespace
 }
 
-func cleanupWordpressBundle(p *porter.TestPorter) {
+func cleanupWordpressBundle(p *porter.TestPorter, targetDependencyName string) {
+
+	// Question: Do need we handle the case of uninstalling all the Dependencies
```

Comment:
This function explicitly cleans up the wordpress bundle in case the test failed, so that we don't leave mysql and/or wordpress installed on your dev machine. Porter handles uninstalling dependencies. The test does call uninstall on wordpress before this function. This is a failsafe in case something bad happens and will attempt to remove test data before the test completes.

So no changes should be necessary here.